web-frontend/src/context/AuthProvider.tsx

```diff
@@ -6,9 +6,7 @@ import axios from "axios";
 const AUTH_KEY = "AuthToken"
 
 
-export const AuthContext = createContext<{
-    token: string | undefined,
-    login: (credentials: {username: string, password: string}) => void }> (
+export const AuthContext = createContext<{ token: string | undefined, login: (credentials: {username: string, password: string}) => void }> (
```

Comment:
Ich fand es in drei Zeilen übersichtlicher.
Es hat sich inhaltlich nichts geändert, oder?

Reply:
Nein, hat sich nicht geändert. Hm, .. unübersichtlicher. Jetzt musst du aber nicht mehr scrollen!11!!